geotags/views.py

```diff
@@ -70,55 +74,56 @@ def geotags_json(request, tag=None):
         #sounds = Sound.public.select_related('geotag').all().exclude(geotag=None)
         q = geoquery % {"join": "", "where": "", "end": ""}
         sounds = Sound.objects.raw(q)
-    return generate_json(sounds)
 
-def geotags_box_json(request):    
+    return generate_bytearray(sounds)
+
+def geotags_box_barray(request):
     box = request.GET.get("box","-180,-90,180,90")
     try:
-        min_lat, min_lon, max_lat, max_lon = box.split(",")  
-        qs = Sound.objects.select_related("geotag").exclude(geotag=None).filter(moderation_state="OK", processing_state="OK")        
+        min_lat, min_lon, max_lat, max_lon = box.split(",")
+        qs = Sound.objects.select_related("geotag").exclude(geotag=None).filter(moderation_state="OK", processing_state="OK")
         if min_lat <= max_lat and min_lon <= max_lon:
             sounds = qs.filter(geotag__lat__range=(min_lat,max_lat)).filter(geotag__lon__range=(min_lon,max_lon))
         elif min_lat > max_lat and min_lon <= max_lon:
             sounds = qs.exclude(geotag__lat__range=(max_lat,min_lat)).filter(geotag__lon__range=(min_lon,max_lon))
         elif min_lat <= max_lat and min_lon > max_lon:
             sounds =qs.filter(geotag__lat__range=(min_lat,max_lat)).exclude(geotag__lon__range=(max_lon,min_lon))
         elif min_lat > max_lat and min_lon > max_lon:
-            sounds = qs.exclude(geotag__lat__range=(max_lat,min_lat)).exclude(geotag__lon__range=(max_lon,min_lon))        
+            sounds = qs.exclude(geotag__lat__range=(max_lat,min_lat)).exclude(geotag__lon__range=(max_lon,min_lon))
 
         sounds_data = [[s.id, s.geotag.lat, s.geotag.lon] for s in sounds]
-        return HttpResponse(json.dumps(sounds_data), content_type="application/json")
+        return generate_bytearray(sounds_data)
```

Comment:
This is related to the comment above. This view is the only one that does not use raw queries. Either this view is refactored to use a raw query (needs careful testing though), or the queryset is annotated with lat and lon properties and passed to `generate_bytearray` directly. Another option is to check if using raw queries is still needed with the version of Django that we use (see comment https://github.com/MTG/freesound/blob/098bfc05e975163aeb3931c648c83e225a31e0fe/geotags/views.py#L34).